src/main/java/de/tum/in/www1/artemis/service/exam/ExamRegistrationService.java

```diff
@@ -274,9 +317,15 @@ public void addAllStudentsOfCourseToExam(Long courseId, Exam exam) {
         userData.put("exam", exam.getTitle());
         for (int i = 0; i < students.size(); i++) {
             var student = students.get(i);
-            if (!exam.getRegisteredUsers().contains(student) && !student.getAuthorities().contains(ADMIN_AUTHORITY)
+            ExamUser registeredExamUserCheck = examUserRepository.findByExamIdAndUserId(exam.getId(), student.getId());
```

Comment:
What is the 'Check' referring to here?

Reply:
This is the exam user that need to be checked if exists in exam. 

I changed the variable name to `examUserToBeChecked`